Installation.md

```diff
@@ -0,0 +1,50 @@
+# Installation Guide for `nteprsm` on Mac
```

Comment:
This is pretty good. Could you please add the content of this file to README.md?